test/fixtures/output/connect/plugin/install/remote-bad-md5.golden

```diff
@@ -0,0 +1,10 @@
+Beginning installation.
+
+Using the only available Confluent Platform installation at "test/fixtures/input/connect/confluent-7.3.3".
+Do you want to install this plugin into "test/fixtures/input/connect/confluent-7.3.3/share/confluent-hub-components"? (y/n): 
```

Comment:
We should test what happens when a user types "n". In this case, I would expect the program to exit and give a suggestion on how they can change the install directory.

Reply:
I already do:
https://github.com/confluentinc/cli/pull/1888/files#diff-ae2a012856605648e0401a8e6d309dfda044924f7a0eb76c52d78a6f08f1d7fd
https://github.com/confluentinc/cli/pull/1888/files#diff-db1052b63cac15718484af3b558f1d9584af0a4ceb57b2f960101ec650ac762c
(`interactive-select-plugin-directory.golden` and `interactive-select-plugin-directory-fail.golden`)

When a user types `n`, it prompts the user to specify a directory. This is the original tool's behavior.